packages/runtime/container-runtime/src/summary/summarizer.ts

```diff
@@ -299,6 +299,10 @@ export class Summarizer extends EventEmitter implements ISummarizer {
 		}
 	}
 
+	public get refreshAndSummarizeLock(): Promise<void> | undefined {
```

Comment:
Where is this used?

Reply:
It's used in tests. We should not add public methods for tests only. There should be other ways to make the test work.

Reply:
Removed